specification/marketplacecatalog/data-plane/Reviews/preview/2023-01-01-preview/reviews.json

```diff
@@ -0,0 +1,405 @@
+{
+  "x-generator": "NSwag v13.18.2.0 (NJsonSchema v10.8.0.0 (Newtonsoft.Json v13.0.0.0))",
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Marketplace Rating And Reviews API",
+    "description": "Microsoft Marketplace Rating And Reviews API",
+    "version": "2023-01-01-preview"
+  },
+  "host": "catalogapi.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/products/{productId}/reviews": {
+      "get": {
+        "tags": [
+          "Reviews Public API"
+        ],
+        "summary": "Get reviews by unique product id",
+        "operationId": "Reviews_GetReviewsByOfferId",
+        "x-ms-examples": {
+          "Facets_Get": {
+            "$ref": "./examples/GetProductReviews.json"
+          }
+        },
+        "produces": [
+          "text/plain",
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "type": "string",
+            "name": "productId",
+            "in": "path",
+            "required": true,
+            "description": "Unique Product Id",
+            "x-nullable": true
+          },
+          {
+            "type": "string",
+            "name": "skipToken",
+            "in": "query",
+            "description": "Skip token to continue getting comments from a given point",
+            "x-nullable": true
+          },
+          {
+            "type": "integer",
+            "name": "pageSize",
+            "in": "query",
+            "description": "Maximum number of reviews to return",
+            "format": "int32",
+            "default": 50,
+            "x-nullable": true
+          },
+          {
+            "type": "string",
+            "name": "api-version",
+            "in": "query",
+            "required": true,
+            "default": "2023-01-01-preview",
+            "x-nullable": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "x-nullable": false,
+            "description": "Returns a list of reviews for the given unique product",
+            "schema": {
+              "$ref": "#/definitions/PageResultOfReviewDto"
+            }
+          },
+          "400": {
+            "x-nullable": false,
+            "description": "In case request is malformed",
+            "schema": {
+              "type": "string"
+            }
+          },
+          "500": {
+            "x-nullable": false,
+            "description": "In case an error encountered",
+            "schema": {
+              "type": "string"
+            }
+          }
+        }
+      }
+    },
+    "/products/{productId}/reviews/{reviewId}/comments": {
```

Comment:
If you're using autorest for your SDK you should mark this route as `x-ms-pageable`, see https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/creating-swagger.md#paging-with-x-ms-pageable